analyses/molecular-subtyping-embryonal/00-embryonal-select-pathology-dx.R

```diff
@@ -18,7 +18,8 @@ output_file <- file.path(root_dir,
 # as we identified in 00-v17-embryonal-select-pathology-dx.Rmd are:
 path_dx_terms<- c(
   "Supratentorial or Spinal Cord PNET",
-  "Embryonal Tumor with Multilayered Rosettes"
+  "Embryonal Tumor with Multilayered Rosettes (ETMR)",
+  "Embryonal Tumor with Multilayered Rosettes (ETMR);High-grade glioma/astrocytoma (WHO grade III/IV)"
```

Comment:
Is _this_ expected?

Reply:
Yes, I added due to a new dx

Reply:
I can revert this back until I get to embryonal. I did it right away so I didn't forget about it, but let me revert that.